src/curses/ui_call_flow.c

```diff
@@ -641,37 +641,19 @@ call_flow_draw_rtp_stream(ui_t *ui, call_flow_arrow_t *arrow, int cline)
     callid = stream->media->msg->call->callid;
     msg_src = stream->media->msg->packet->src;
     msg_dst = stream->media->msg->packet->dst;
-    stream_src = stream->src;
-    stream_src.port = 0;
-    stream_dst = stream->dst;
-    stream_dst.port = 0;
-
-    // Get origin column for this stream.
-    // If we share the same Address from its setup SIP packet, use that column instead.
-    if (!strcmp(stream->src.ip, msg_src.ip)) {
-        column1 = call_flow_column_get(ui, callid, msg_src);
-    } else if (!strcmp(stream->src.ip, msg_dst.ip)) {
-        column1 = call_flow_column_get(ui, callid, msg_dst);
-    } else {
-        column1 = call_flow_column_get(ui, 0, stream_src);
-    }
 
-    // Get destination column for this stream.
-    // If we share the same Address from its setup SIP packet, use that column instead.
-    if (!strcmp(stream->dst.ip, msg_dst.ip)) {
-        column2 = call_flow_column_get(ui, callid, msg_dst);
-    } else if (!strcmp(stream->dst.ip, msg_src.ip)) {
-        column2 = call_flow_column_get(ui, callid, msg_src);
-    } else {
-        column2 = call_flow_column_get(ui, 0, stream_dst);
-    }
+    // Get call inititator source and destination address
+    src = stream->media->msg->call->cstart_msg->packet->src;
+    dst = stream->media->msg->call->cstart_msg->packet->dst;
```

Comment:
My rationale, which very probably is wrong, was:

- A call is composed by two endpoints.
- The direction of both endpoints is known by means of `src` and `dst` address:port of the initial SIP message packet.
- A stream has a `media` associated which represent the SDP 'm=' line  containing its description, which offerer is known by means of the the `src` and `dst` address:port of the SIP message packet containing the media description.
-  Given the previous facts, we know who (caller, callee) offered the media description for the stream we are analysing and thus we can accurately draw the RTP stream accurately. 

This simplified rationale made me remove the 'reverse' streams, since it was associating the opposite media description to the stream. (among other reasons)